model_zoo/census_model_sqlflow/wide_and_deep/transform_ops.py

```diff
@@ -0,0 +1,74 @@
+from enum import Enum
+
+
+class TransformOpType(Enum):
+    HASH = (1,)
+    BUCKETIZE = (2,)
+    LOOKUP = (3,)
+    EMBEDDING = (4,)
+    CONCAT = (5,)
+    ARRAY = 6
+
+
+class TransformOp(object):
+    def __init__(self, name, input, output):
+        self.name = name
```

Comment:
Can the `name` and `output` be optional? We can set a default value according to input for `name` and `output`. For example, the `name` and `output` can be the same as `input` because the `input` is unique.

Reply:
I prefer to decouple input and output name. 
For one input, we may apply two different transform actions and get two different outputs.
BTW, the output name is auto-generated.

Reply:
I wonder whether we will provide users with those `TransformOP`s to develop preprocessing logic in their model definition. If it is, the required arguments are as less as possible. So, it may be better to auto-generate `output` and `name`.

Reply:
In my original opinion, I won't publish this `TransformOP` to users.
If you are planning to do so, you can simplify based on this.